addons/web/static/src/views/kanban/column_progress.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="web.ColumnProgress" owl="1">
+        <div class="o_column_progress progress bg-300 w-75">
+            <t t-foreach="props.group.progressBars" t-as="progressBar" t-key="progressBar.value">
```

Comment:
the t-key here looks strange. not sure if it could happen, but what if there is two progressbars with the same value? I suggest you find some unique id, either an index if we know it can not be reordered, or maybe some kind of unique string, such as the type